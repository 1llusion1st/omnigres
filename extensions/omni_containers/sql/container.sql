-- Creating container
SELECT count(*) FROM pg_catalog.pg_tables WHERE tablename = 'container_test';
SET client_min_messages TO WARNING;
SELECT true FROM omni_containers.docker_container_create('governmentpaas/psql', cmd => $$psql -c 'CREATE TABLE container_test ();'$$,
                                                      wait => true, pull => true);
SET client_min_messages TO NOTICE;
SELECT count(*) FROM pg_catalog.pg_tables WHERE tablename = 'container_test';

-- Creating a container that can't be found
SELECT true FROM omni_containers.docker_container_create('cannotbefound');