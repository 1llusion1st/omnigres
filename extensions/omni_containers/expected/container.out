-- Creating container
SELECT count(*) FROM pg_catalog.pg_tables WHERE tablename = 'container_test';
 count 
-------
     0
(1 row)

SET client_min_messages TO WARNING;
SELECT true FROM omni_containers.docker_container_create('postgres', cmd => $$psql -c 'CREATE TABLE container_test ();'$$,
                                                      wait => true, pull => true);
 ?column? 
----------
 t
(1 row)

SET client_min_messages TO NOTICE;
SELECT count(*) FROM pg_catalog.pg_tables WHERE tablename = 'container_test';
 count 
-------
     1
(1 row)

-- Creating a container that can't be found
SELECT true FROM omni_containers.docker_container_create('cannotbefound');
ERROR:  Docker image not found
DETAIL:  cannotbefound
